.circleci/config.yml

```diff
@@ -186,11 +186,15 @@ workflows:
           context:
             - org-global
           requires:
-              - test
+            - test
   production:
     when:
       and:
-        - equal: [ https://github.com/Yonomi/yonomi-sdk-dart, << pipeline.project.git_url >> ]
+        - equal:
```

Comment:
does the format change anything about the `yml` doc?  haven't messed with circle configs, so mostly wanted to double-check.

Reply:
Seems valid based on validation but I changed it back because I prefer it to be more compact.